app/assets/stylesheets/juntos_bootstrap/components/_cards.scss

```diff
@@ -0,0 +1,153 @@
+.card,
+.card-notification {
+  border: 1px solid $border-strong;
+  padding: 18px;
+}
+
+.card-notification {
+  background-color: $blue-note;
+  border-color: $blue-note;
+  border-radius: 5px;
+  box-shadow: $box-shadow-color-darken 0 1px 4px 0;
+  color: $white;
+  margin-right: 20px;
+  margin-top: 20px;
+  max-height: rem-calc(60);
+  position: fixed;
+  right: 0;
+  top: 0;
+  width: 30%;
+  z-index: 20;
+}
+
+.card-project-description {
+  padding: 8px;
+
+  ._1 {
+    padding-bottom: 12px;
+    padding-left: 12px;
+    padding-right: 12px;
+  }
+
+  ._3 {
+    height: 165px;
+    padding: 13px 14px 20px;
+  }
+}
+
+.card-project-description._2 {
```

Comment:
Some reason not to use `&._2`?

Also, another think that made me confused: why is there a nested `._1` (implying `.card-project-description ._1`) and a non-nested `.card-project-description._2`?